react/week3/todo/src/App.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from "react";
+import "./App.css";
+import Header from "./components/Header";
+import Timer from "./components/Timer";
+import TodoForm from "./components/TodoForm";
+import TodoList from "./components/TodoList";
+
+function App() {
+  const [todos, setTodos] = useState([]);
+  const [isLoaded, setIsLoaded] = useState("false");
+
+  const url =
+    "https://gist.githubusercontent.com/benna100/391eee7a119b50bd2c5960ab51622532/raw";
+  async function fetchingData() {
+    const result = await fetch(url);
+    const todos = await result.json();
+    setTodos(todos);
+    setIsLoaded(false);
```

Comment:
Maybe I'm missing something but from what I see `isLoaded` never has the value of true, since the default state is `false` and you assign `false` to it here as well.